code/modules/events/crystal_event.dm

```diff
@@ -109,14 +109,28 @@ This section is for the event controller
 	for(var/i in 1 to 6)
 		spawn_anomaly(spawners)
 
-	addtimer(CALLBACK(src, .proc/more_portals, GLOB.crystal_invasion_waves[wave_name]), 15 MINUTES)
+	var/list/crystal_spawner_turfs = list()
+	for(var/turf in RANGE_TURFS(6, dest_crystal.loc))
+		if(!isopenturf(turf) || isspaceturf(turf))
+			continue
+		var/turf/floor = turf
+		crystal_spawner_turfs += floor
+	for(var/i = 0, i < 6, i++)
```

Comment:
```suggestion
	for(var/i in 1 to 6)
```
Favor the list loop over the C one whenever you have no need to modify the index.